lib/matplotlib/axes/_base.py

```diff
@@ -17,6 +17,7 @@
 from matplotlib import cbook
 from matplotlib.cbook import (_check_1d, _string_to_bool, iterable,
                               index_of, get_label)
+from collections import OrderedDict
```

Comment:
I know we don't follow this much yet, but can you move stdlib imports above the local (matplotlib) imports?